### ISSUE TITLE
Laser: Init from File

### ISSUE THREAD

**Issue by ax3l** [ComputationalRadiationPhysics/picongpu#1503]
Allow to initialize the laser on a boundary (like now) from a hdf5 file with planes(t) instead of from a in-code formula.

Allows scripting new laser profiles quickly: #1502

**Comment by wmles** [comment]
I agree that this is a good idea, would have made my project much faster. Unfortunately, I dont think I can just implement it ... probably not even help on the way, can I?

**Comment by ax3l** [comment]
We have implemented similar setups of density inits already. If you are willing to make this your project & dig a bit deeper we can certainly assist you and guide you along the way.
